ecosystem/indexer-server/typescript/src/api/openapi.yaml

```diff
@@ -0,0 +1,3333 @@
+openapi: 3.0.3
```

Comment:
we can just keep this one and delete the one under indexer-server/doc

Reply:
I think we should keep the indexer-server/doc and this one. This /api/openapi.yaml was autogenerated from the indexer-server/doc openapi.yaml but includes a few more fields for controllers (they're not identical).